### ISSUE TITLE
Non-public list archive (Google Groups)

### ISSUE THREAD

**Issue by paulmenzel** [Azure/SONiC#692]
Trying to look at [the mailing list archive][1], it says, I have to be logged in to view. Could the archives please be made public for people without a Google account.

[1]: https://groups.google.com/forum/#!forum/sonicproject

**Comment by stolsma** [comment]
Yep, have the same problem... Could someone change the security setting of the archive(s)?
